databricks/koalas/tests/test_dataframe.py

```diff
@@ -1009,6 +1010,11 @@ def test_join(self):
 
         self.assert_eq(join_pdf, join_kdf)
 
+        # join with duplicated columns in Series and DataFrame
+        with self.assertRaisesRegex(ValueError,
+                                    "columns overlap but no suffix specified"):
+            kdf1.join(ks1, how='outer')
+            kdf1.join(kdf2, how='outer')
```

Comment:
Actually this only tests the first line.

Reply:
Sorry. I will fix it.

Reply:
Ah yes. Sorry for missing this one  will take a closer look next time @ueshin